### DISCUSSION TITLE
¿Where are the source code files?

### DISCUSSION THREAD

**Discussion by akihiro13Dev** [vuetifyjs/vuetify#14609] [Q&A] [answered]
## A little question from a beginner
Hi, I'm looking contribute in this great project. But I do not know where are the source files. I found a few possible directories.
One contains files written with JavaScript, and other contains file written with TypeScript. ¿Whitch can be the correct directory?

-  **First directory:** _/vuetify/lib/components/_

- **The Second directory:** _/vuetify/src/components/_

## Then
Thank a lot. I hope my problem is understood.

**Comment by jacekkarczmarczyk** [comment, marked as answer]
https://github.com/vuetifyjs/vuetify/tree/master/packages/vuetify/src
